modules/apps/batch-planner/batch-planner-web/test/setup.js

```diff
@@ -25,23 +25,24 @@ window.Liferay.Language = {
 };
 
 window.Liferay.ThemeDisplay = {
-	getBCP47LanguageId: () => 'en-US',
+	...window.Liferay.ThemeDisplay,
 	getPathThemeImages: () => '/assets',
```

Comment:
It seems like this should also exist in our default ThemeDisplay mock, that way we wouldn't even need to specify this here.